docs/StardustDocs/topics/guides/Guide-for-backend-SQL-developers.md

```diff
@@ -0,0 +1,225 @@
+# Kotlin DataFrame for SQL & Backend Developers
+
+<web-summary>
+Quickly transition from SQL to Kotlin DataFrame: load your datasets, perform essential transformations, and visualize your results — directly within a Kotlin Notebook.
+</web-summary>
+
+<card-summary>
+Switching from SQL? Kotlin DataFrame makes it easy to load, process, analyze, and visualize your data — fully interactive and type-safe!
+</card-summary>
+
+<link-summary>
+Explore Kotlin DataFrame as a SQL or ORM user: read your data, transform columns, group or join tables, and build insightful visualizations with Kotlin Notebook.
+</link-summary>
+
+This guide helps Kotlin backend developers with SQL experience quickly adapt to **Kotlin DataFrame**, mapping familiar SQL and ORM operations to DataFrame concepts.
+
+We recommend starting with [**Kotlin Notebook**](SetupKotlinNotebook.md) — an IDE-integrated tool similar to Jupyter Notebook.  
+
+It lets you explore data interactively, render DataFrames, create plots, and use all your IDE features within the JVM ecosystem.
+
+If you plan to work on a Gradle project without a notebook, we recommend installing the library together with our [**experimental Kotlin compiler plugin**](Compiler-Plugin.md) (available since version 2.2.*).  
+This plugin generates type-safe schemas at compile time, tracking schema changes throughout your data pipeline.
+
+<!---IMPORT org.jetbrains.kotlinx.dataframe.samples.guides.QuickStartGuide-->
+
+## Quick Setup
+
+To start working with Kotlin DataFrame in a Kotlin Notebook, run the cell with the next code:
+
+```kotlin
+%useLatestDescriptors
+%use dataframe
+```
+
+This will load all necessary DataFrame dependencies (of the latest stable version) and all imports, as well as DataFrame
+rendering. Learn more [here](SetupKotlinNotebook.md#integrate-kotlin-dataframe).
+
+---
+
+## 1. What is a DataFrame?
```

Comment:
Please follow here (and everywhere in the guide) our spelling conventions;
https://kotlin.github.io/dataframe/spellingconventions.html

`DataFrame` as object/type/class should be in backtics.
"dataframe" as a concept of tabular data (both abstract or concrete) should be written in lowercase.